src/box/vy_run.c

```diff
@@ -969,44 +998,111 @@ static int
 vy_page_read_cb(struct cbus_call_msg *base)
 {
 	struct vy_page_read_task *task = (struct vy_page_read_task *)base;
-	ZSTD_DStream *zdctx = vy_env_get_zdctx(task->run->env);
+	ZSTD_DStream *zdctx = vy_env_get_zdctx(task->run->run_env);
 	if (zdctx == NULL)
 		return -1;
 	if (vy_page_read(task->page, task->page_info, task->run, zdctx) != 0)
 		return -1;
 	if (task->key.stmt != NULL) {
 		task->pos_in_page = vy_page_find_key(task->page, task->key,
-						     task->cmp_def, task->format,
+						     task->cmp_def,
+						     task->format,
 						     task->iterator_type,
 						     &task->equal_found);
 	}
 	return 0;
 }
 
 /**
- * Increase memory-stored page's reference counter.
+ * Delete fixed percentage of pages contained in page cache's LRU list -
+ * starting from the least recently used one and further towards the head
+ * of the list. At least one page is deleted in any case, except in the case
+ * of empty list.
+ */
+static inline void										//temp/ maybe should add error codes (what errors?..)
```

Comment:
What?

Reply:
Never mind, already removed this comment